config/title_screens/README.txt

```diff
@@ -1,7 +1,10 @@
-The enclosed /images folder holds the image files used as the title screen for the game. All common formats such as PNG, JPG, and GIF are supported.
-Byond's DMI format is also supported, but if you use a DMI only include one image per file and do not give it an icon_state (the text label below the image).
+The enclosed /images folder holds the image files used as the title screen for the game.
+
+Specification:
+Formats: PNG, JPG, GIF, DMI*
+Dimensions: 544x480
```

Comment:
its still incorrect widescreen is not 544

Reply:
and, e-mail reply did not atatch it to the thread >_<

The title screen is overlayed on a special area of the centcom map. the dimensions of this special area determine the pixel dimensions of the title card. Current standard for TG for a tile is 32x32.  this section of the map is 17 tiles wide by 15 tiles tall. thus 544x480

Reply:
well that explains why that turf is out of bounds in strong dmm  *facepalms*